tests/backstop_data/engine_scripts/puppet/mouse-events-helper.js

```diff
@@ -2,7 +2,7 @@
 
 const Utility = require('./utility.js');
 
-module.exports = async (page, scenario, viewport) => {
+module.exports = async (page, scenario, viewport, dontWait) => {
```

Comment:
too many parameters. Let's use options if possible. And instead of don't wait why not a `{ wait: 0 }` with the default being `1000` ?